harness/determined/cli/command.py

```diff
@@ -1,8 +1,9 @@
 import base64
 import json
+import operator
```

Comment:
Ugh. I hate that this file is called `command.py` when we have a `det command` that does not execute from this file.

Since you've already put the work into figuring out what this file _actually_ does, could you please bonus give it a short, module-level docstring?

Reply:
Or as a super-bonus, rename the dang file?

Reply:
ok so `remote.py` can become `command.py`; and this becomes `ntsc.py`?..

Reply:
I love the result. I think if it was me who was doing it, I'd do it exactly the same as you've done it here and bundle both the renaming and the functionality into the same PR. But I'd do it with a vague sense I should instead break the PR into move & functionality rather than all at once.

There's no way I'm going to ask you to do a thing I wouldn't have chosen to do, myself. But I also feel like I wouldn't be doing my job as a reviewer without prompting you to make an explicit decision about what to do, yourself.

Reply:
Also, thanks for making the change. Wasn't your fault it got this way, but think having made the change makes the CLI a little less gross.

Reply:
in contrast with the reformat, `git mv` does not really mess up `git blame`. I think this is enough scope creep already.

Reply:
Proposal here to rename `ntsc.py` to `_ntsc.py`. I know the CLI shouldn't be public, anyway, but this small change helps make it slightly more obvious that `ntsc` is a shared library of utilities rather than a command.

Reply:
is it worth doing these big style guide changes piecemeal? it'd only make this directory seem more chaotic.